Makefile

```
@@ -286,11 +299,15 @@ endif
 ifdef TRAVIS_TAG
 	docker tag $(DOCKER_REPOSITORY) $(DOCKER_REPOSITORY):$(TRAVIS_TAG:v%=%)
 endif
+else
+	@echo skipping $(DOCKER_FILE)
+endif
 endif
 endif
 
 docker-push:
 ifdef DOCKER
+ifdef BUILD_SERVICE:
```

```suggestion
ifdef BUILD_SERVICE
```
Was the `:` a typo? But if it was a typo, it shouldn't have ever matched, and so should have never worked?

---

Definitely typo, I never tested the push